services/ingestion/block_tracking_subscriber.go

```diff
@@ -369,6 +379,11 @@ func (r *RPCBlockTrackingSubscriber) getEventsByType(
 
 	// The `EVM.BlockExecuted` event should be present for every Flow block.
 	if strings.Contains(eventType, string(events.EventTypeBlockExecuted)) && len(event.Events) != 1 {
+		// The spork root block has no transactions, and therefore no EVM blocks.
```

Comment:
```bash
// The spork root block has no transactions
```
Does this mean it doesn't even have the system chunk transaction? If so, we could possibly move this `if` statement at the top of the method's body, as we know that the spork root block will never contain any events of `eventType`. That's very minor though, so feel free to leave it as is.

Reply:
yea, there are no system tx either. I put it here to avoid doing a check on every block for the special case that happens one block per year.

Reply:
Ah right :+1: Sounds good then.